### PR TITLE
Cosmetic changes to fix warning from code analysis.

### PR DESCRIPTION
**Pull request by riverszhang89**
Use of freed values, undefined externs, etc.

### CODE REVIEW

File: bdb/bdb_verify.c [hunk 1]
```diff
@@ -815,7 +815,7 @@ static int bdb_verify_ll(
 
                         if (rc == 0) {
                             realblobsz[blobno] = dbt_blob_data.size;
-                            if (blobsizes[blobno] == -1 && rc != DB_NOTFOUND) {
```
Comment (nirbhayc) [inline bdb/bdb_verify.c]: What was the warning here?

Comment (riverszhang89) [reply bdb/bdb_verify.c]: `rc != DB_NOTFOUND` will always to true as `if (rc == 0)` guarantees it.

Comment (nirbhayc) [reply bdb/bdb_verify.c]: right

**Comment by riverszhang89** [comment]
Thank you for the review Nirbhay!

**Pull request merged**